.changeset/fresh-beans-call.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': patch
+---
+
+Add ability to customize Read More destination for MissingAnnotationEmptyState component
```

Comment:
Let's mention the new prop that this was added through, and wrap code names up in `` ` `` to make them look nice :grin: